invoices/helpers_test.go

```diff
@@ -0,0 +1,209 @@
+package invoices
```

Comment:
I think we generally call this file `utils_test.go`

Reply:
Moved to `test_utils_test.go` as discussed offline.